docs/supported-libraries.md

```diff
@@ -15,7 +15,7 @@ or [contributing](../CONTRIBUTING.md).
 
 ## Libraries / Frameworks
 
-These are the supported libraries and frameworks:
+These are the supported libraries and frameworks (sort alphabetically):
```

Comment:
> I'm not sure this is needed

In preview PR, @laurit told me need to add libraries in alphabetical order. I think we need to add relevant note, otherwise it is easy to ignore.
![image](https://github.com/open-telemetry/opentelemetry-java-instrumentation/assets/25423978/ac48fb5d-3e81-46e8-975a-d0d53cda6f7a)

Reply:
I'd still suggest removing here, prioritizing readers of the page (as opposed to editors of the page)

Reply:
> I'm not sure this is needed

I think that adding a related note does not actually affect readers' reading. Instead, it allows readers to understand its sorting rules, which makes it easier for them to retrieve information:) don't you think so? If after saying this you still feel it’s not necessary, then I will delete it？

Reply:
> I'd still suggest removing here, prioritizing readers of the page (as opposed to editors of the page)

I think that adding a related note does not actually affect readers' reading. Instead, it allows readers to understand its sorting rules, which makes it easier for them to retrieve information:) don't you think so? If after saying this you still feel it’s not necessary, then I will delete it？

Reply:
> my preference is still to remove it here 😄

ok, I removed it now.